### ISSUE TITLE
pipeop input task mismatch when using AutoFSelector

### ISSUE THREAD

**Issue by danieldimatteo** [mlr-org/mlr3fselect#25]
Thanks in advance for your time and attention,

I'm attempting to use AutoFSelector on a graph learner containing an imputemedian pipeop, and the following error is thrown:

> Error: Input task during prediction of imputemedian does not match input task during training.

Code to reproduce problem:

```
library("paradox")
library("mlr3fselect")

learner = lrn("classif.rpart", predict_type = "prob")
graph = po("imputemedian") %>>% learner
gl = GraphLearner$new(graph)

at = AutoFSelector$new(
  learner = gl,
  resampling = rsmp("holdout"),
  measure = msr("classif.logloss"),
  terminator = trm("evals", n_evals = 10),
  fselector = fs("random_search")
)

grid = benchmark_grid(
  task = tsk("pima"),
  learner = list(at, lrn("classif.rpart", predict_type = "prob")),
  resampling = rsmp("cv", folds = 3)
)

bmr = benchmark(grid, store_models = TRUE)
bmr$aggregate(msrs(c("classif.logloss", "time_train")))
```

**Comment by be-marc** [comment]
Thanks for reporting this bug. You can install a fixed version with:

```
remotes::install_github("mlr-org/mlr3fselect@AutoFSelect_PipeOp")
```

We will discus tomorrow if all cases are covered by this. You can expect an updated CRAN version in a few days.

**Comment by be-marc** [comment]
Fixed in version 0.4.1. Released on CRAN.
